R/adl.R

```diff
@@ -122,4 +121,160 @@ adl_fun <- function (ADL_01, ADL_02, ADL_03, ADL_04, ADL_05) {
            "NA(b)"
            )
 
-}
\ No newline at end of file
+}
+
+#' @title The number of activities of daily living tasks that require help.
+#'
+#' @description A 6 category variable (ADL_score_5) representing the number of
+#'  activities of daily living tasks that require help. The following five ADL
+#'  variables are used. All variables are common across CCHS cycles from 2001
+#'  to 2014:
+#' \enumerate{
+#'   \item ADL_01 - Needs help preparing meals
+#'   \item ADL_02 - Needs help getting to appointments/errands
+#'   \item ADL_03 - Needs help doing housework
+#'   \item ADL_04 - Needs help doing personal care
+#'   \item ADL_05 - Needs help moving inside house
+#' }
+#' There are two categories to represent "missing" and "not applicable". 
```

Comment:
I know you mention the tagged nas in  the return but this sentance feels incomplete. I suggest adding some reference to the tagged na here as well.